docs/concepts/portal/dashboard.md

```diff
@@ -0,0 +1,14 @@
+---
+title: " Dashboard Page Reference"
+linkTitle: "Dashboard Page Reference"
+---
+
+
+The Dashboard gives you an unified and summarised view of the application you have deployed. Top view all of your application scroll the Dashboard page. If there are any persistent volumes in error this will be indicated on the dashboard as a red side line next to the application name and will also give you the number of affected volumes 
+
+Your deployed application can be one of the following types:
+* Replica - <add_description_here>
+* StatefulSet - <add_description_here>
```

Comment:
A [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) represents a stateful application that both manages one or more pods, ensures that they are running at any given time and provides certain guarantees about the order and uniqueness of the pods.

Reply:
Thanks,, useful description - will include